package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "azure-samples-for-js-management",
+  "version": "1.0.0",
+  "description": "Azure Samples For JS management SDK",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint . --ext .ts --ignore-pattern node_modules/ --ignore-pattern dist/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Azure-Samples/azure-samples-js-management.git"
+  },
+  "author": "",

```

Comment:
Do we have an alias for JS SDK?

Reply:
@nickzhums  Could you take a look at this question ?

Reply:
do not think so